js/contextMenus.js

```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
         CommonMenu.separatorMenuItem,
         CommonMenu.importBookmarksMenuItem()
       ]
-    }, {
-      label: locale.translation('bravery'),
-      submenu: [
```

Comment:
we should still prefer the Bravery menu. 

CC @bradleyrichter which order would you like it in?